flocker/volume/test/test_httpapi.py

```diff
@@ -15,26 +15,27 @@
 from ...restapi.testtools import (
     buildIntegrationTests, loads, goodResult)
 
-from ..httpapi import DatasetAPIUser, create_api_service
+from ..httpapi import DatasetAPIUserV1, create_api_service
+from ... import __version__
 
 
 class APITestsMixin(object):
     """
     Integration tests for the Dataset Manager API.
     """
-    def test_noop(self):
+    def test_version(self):
```

Comment:
If following the HybridCluster guide, this should perhaps be in `class VersionTestsMixin` or similar.

> Tests should be grouped together by endpoint and purpose. For example, all of the tests for the website creation endpoint should be defined in one class.